src/api.js

```diff
@@ -1,60 +1,44 @@
 function handleError(res) {
+  if(!res.ok) {
     throw new Error(`HTTP Error: ${res.status} -- Please try again later`)
   }
+  return res.json()
+}
   
-  const getTrendingAlbums = () => {
-    return fetch('https://api.discogs.com/database/search?type=master&format=vinyl&key=GimREdkHlKcSjALMSwEP&secret=RZbpExNDRyTdbTAaiVxiJpiYgOcydrMJ&page=1&per_page=5&sort=hot')
-        .then(res => {
-            if (res.ok) {
-                return res.json()
-            }
-            handleError(res)
-        })
-        
-  }
+const getTrendingAlbums = () => {
+  return fetch('https://api.discogs.com/database/search?type=master&format=vinyl&key=GimREdkHlKcSjALMSwEP&secret=RZbpExNDRyTdbTAaiVxiJpiYgOcydrMJ&page=1&per_page=5&sort=hot')
+      .then(res => handleError(res))
+}
   
-  const getRecommendedAlbums = (albumID) => {
-    return fetch(`https://api.discogs.com/masters/${albumID}?key=GimREdkHlKcSjALMSwEP&secret=RZbpExNDRyTdbTAaiVxiJpiYgOcydrMJ`)
-        .then(res => {
-          if (res.ok) {
-            return res.json()
-          }
-          handleError(res)
-        })
-        .then(data => fetch(`https://api.discogs.com/database/search?type=master&format=vinyl&key=GimREdkHlKcSjALMSwEP&secret=RZbpExNDRyTdbTAaiVxiJpiYgOcydrMJ&page=1&per_page=5&genre=${data.genres[0]}`))
-        .then(res => {
-          if (res.ok) {
-            return res.json()
-          }
-          handleError(res)
-        })
-  }
```

Comment:
Can we leave this in because I think I still need to consume it for my ticket?

Reply:
You can still use it, it was just renamed and you can add what you have on line 24 & 25 after calling the function. 3 parts of the app need what we're fetching on line 17, so I think it makes more sense to take out line 24 & 25 and just use it when you invoke the function.

Reply:
ah okay i see now thanks!